### PR TITLE
Update homepage to reference docs

### PR DESCRIPTION
**Pull request by DavidLemayian**
This is instead of the old Github repo.

### CODE REVIEW

File: wazimap/templates/homepage.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h1 class="article-header">About {{ WAZIMAP.name }}</h1>
             <p>Read more about <a href="/about">who we are and why we built Wazimap</a>.
 
             <h2 class="column-full subhead">Shoulders of Giants</h2>
-            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can <a href="https://github.com/Code4SA/censusreporter/">fork Wazimap on Github</a> and build one for your country.</p>
+            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can build one for your country by following the documentation <a href="https://wazimap.readthedocs.org/en/latest/index.html">here</a>.</p>
```
Comment (longhotsummer) [inline wazimap/templates/homepage.html]: I'm not a fan of hyperlinks that say "here". How about: "You can [build one for your country]."

Comment (DavidLemayian) [inline wazimap/templates/homepage.html]: Completely agree. I figured the entire website needs a content overhaul but thought I'd just point this out.

**Comment by DavidLemayian** [comment]
@longhotsummer Ok, fixed and squashed. :)

**Comment by longhotsummer** [comment]
:+1:

**Comment by longhotsummer** [comment]
I'll prepare a new release in a few days. I hope to add the map to the homepage as part of the same release.

**Comment by DavidLemayian** [comment]
@longhotsummer Great! Can't wait..

Actually couldn't because of some changes you had made so went github repo pulling instead of via PyPi. :)

**Pull request merged**